src/Microsoft.AspNet.Mvc.Core/UnitTestability/MvcControllerUnitTestHelper.cs

```diff
@@ -0,0 +1,277 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.WebSockets;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Http.Core.Collections;
+using Microsoft.AspNet.Http.Security;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
+
+// Not sure if this is the right location/namespace for this..
+namespace Microsoft.AspNet.Mvc.Core
+{
+	public class MvcControllerUnitTestHelper
```

Comment:
Agree. It was static before, but while I'm trying to subclass UrlHelper, I made it a regular type and forgot to reverse. Thanks.